### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by ConnyOnny** [hfiguiere/leftpad-rs#7]
The API is not very complicated nor prone to changes. I suggest putting a 1.0 version on this, as it communicates to developers that this crate can be used without the risk of any breaking API changes.
See http://semver.org for a motivation.

**Comment by hfiguiere** [comment]
You know this crate was written as a joke?

**Comment by ConnyOnny** [comment]
Yes. But it doesn't mean it won't be used. Rust also targets developers coming from scripting languages like JavaScript. If they find a package like this, they recognize the name and expect the package to be in good shape.
Note that there is another crate for this purpose/making the same joke: https://crates.io/crates/left-pad
